rct_optimizations/test/extrinsic_multi_static_camera_utest.cpp

```diff
@@ -46,83 +34,116 @@ void printResults(const rct_optimizations::ExtrinsicMultiStaticCameraMovingTarge
   }
 }
 
-void addObservations(const Eigen::Vector3d& axis,
-                     const std::vector<Eigen::Vector3d> &target_points,
-                     const std::vector<Eigen::Isometry3d> &base_to_camera,
-                     const Eigen::Isometry3d &wrist_to_target,
-                     rct_optimizations::ExtrinsicMultiStaticCameraMovingTargetProblem &problem_def)
+struct Observations
 {
-  for (int i = -10; i <= 10; ++i)
-  {
-    Eigen::Isometry3d base_to_wrist;
+  std::vector<Eigen::Isometry3d> wrist_poses;
+  std::vector<CorrespondenceSet> correspondences;
+};
+
+Observations addObservations(const test::Target &target,
+                             const test::Camera &camera,
+                             const Eigen::Isometry3d &base_to_camera,
+                             const Eigen::Isometry3d &wrist_to_target,
+                             const Eigen::Vector3d &axis)
+{
+  const int n = 20;
```

Comment:
Could we make `n` configurable?

Reply:
Yes, but it doesn't make sense to do it here because it is within the unit test. Ultimately I want to move this function to `observation_creator.h`, where we would add the number of observations as an input parameter. I was hoping to do this after the restructure proposed in #38 and a PR from @colin-lewis-19 coming this week to formalize the generation of poses.